src/Panels/ChatLogPanel.cs

```diff
@@ -243,14 +297,14 @@ private void OnChatKeyDown(UIComponent component, UIKeyEventParameter eventParam
                 }
 
                 // If not connected to a server / hosting a server, tell the user and return
-                if (MultiplayerManager.Instance.CurrentRole == MultiplayerRole.None)
-                {
-                    PrintGameMessage(MessageType.Warning, "You can only use the chat feature when hosting or connected.");
-                    return;
-                }
+                //if (MultiplayerManager.Instance.CurrentRole == MultiplayerRole.None)
```

Comment:
Commented out the need for the user not being able to input if the server is not running. Easy for testing and did state the user is local.